README.md

```diff
@@ -52,6 +52,14 @@ Navigate to [localhost:8080](http://localhost:8080/) to see the API
 
 ## Usage
```

Comment:
Is there a way we can transfer all the information that's in this `## Usage` section into the swagger site?

That way the main README.md be a lot smaller.

Reply:
To my understanding, swagger just allows you to automate the doc itself, so this is supposed to assist the website as a "docs" tab. So when we create the main igboapi.com website, we can add this to that tab and update the elements to match the CSS for the original template.

Reply:
I see what you mean, so once we have a dedicated docs site we can just change the styles that swagger automatically uses so that it matches the look of our site.